tests/functional/step_defs/test_chart_src_with_report.py

```diff
@@ -73,3 +76,14 @@ def index_yaml_is_updated_with_new_entry(workflow_test):
 def release_is_published(workflow_test):
     """a release is published with corresponding report and chart tarball."""
     workflow_test.check_release_result()
+
+@then("the pull request is not merged")
```

Comment:
These methods appear in several files which I would like to get away from. If it means we have one big feature file I do not object to that although maybe one python routine could import several features.

Reply:
Merging this PR, we can discuss these improvements and initiate another PR.